core/src/test/scala/org/http4s/UrlFormSpec.scala

```diff
@@ -70,7 +66,7 @@ class UrlFormSpec extends Http4sSpec with ScalaCheck {
       UrlForm(Map("key" -> Seq("a", "b", "c"))).getFirstOrElse("notFound", "d") must_== "d"
     }
 
-    "construct consistently from kv-pairs or and Map[String, Seq[String]]" in check {
+    "construct consistently from kv-pairs or and Map[String, Seq[String]]" in prop {
```

Comment:
Weird, had to do a `clean test:compile` before getting this on my local machine. It is due to the `NonEmptyList` (i.e. it works fine if I change it to a `List`), but I am having a real hard time figuring out why it suddenly is not working anymore.